chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -1425,6 +1446,15 @@ def get_features(self):
         return rf
 
 
+@directory.register
+class UV21ProGPS(UV17ProGPS):
+    VENDOR = "Baofeng"
+    MODEL = "UV-21ProGPS"
+
+    VALID_BANDS = UV21ProV2.VALID_BANDS
+    MODES = UV21ProV2.MODES
```

Comment:
I haven't looked at what the base class uses but there are a lot of variants in here already that differ by very little if anything. Is this really a unique combination?